www/index.html

```diff
@@ -60,13 +110,9 @@
         <a class="tab-item" href="#/reservations">
             Reservations
         </a>
-        <a class="tab-item" href="#/manage">
```

Comment:
This is just temporary, for some reason I kept seeing this page overriding the tab bar. I'll fix it before I created a PR to be merged.